docs/ingestion/kafka-ingestion.md

```diff
@@ -413,10 +421,13 @@ This query returns:
 |------------------|-----------|-----------------|-------------|
 |`development`|`wiki-edit`|`1680795276351`|`wiki-edits`|
 
-### Tuning configuration
+### `tuningConfig`
+
+Specify tuning properties in a `tuningConfig` object at the top level of an ingestion spec.
```

Comment:
Since this is not a tutorial, I think starting this sentence with "you" improves readability. 
```suggestion
You specify tuning properties in a `tuningConfig` object at the top level of an ingestion spec.
```